lib/utils/svgTemplates.mjs

```diff
@@ -0,0 +1,43 @@
+/**
+@module /utils/svgTemplates
+*/
+
+/**
+@function svg_templates
+@async
+
+@description
+The method parses the templates object param. Templates which are not yet available as mapp.utils.svgSymbols.templates{} will fetched and stored for use in feature style methods.
+
+The svgTemplates method will resolve once all templates have been parsed and loaded.
+
+@param {object} templates An object with svg_template properties to load.
+
+@returns {Promise<void>}
+*/
+export default async function svgTemplates(templates){
+
+  if (!Object.keys(templates)) return;
+
```

Comment:
This is going to be an issue with undefined or null templates.

Going to add an additional check if templates are undefined or null.

```js
  if (!templates || !Object.keys(templates).length) return;
```

Reply:
Not sure I fully understand the issue but happy to expand the conditional check.

Reply:
So when a workspace has sync plugins but no svgTemplates in the workspace it would hit the svgTemplates function with (undefined/null) because it's not part of the workspace. So just like we have that check in the loadPlugins utility we will also expand the check :) If not clear I can show can example tomorrow in the stand up :)

Reply:
Good catch.